App/react-game/src/Game/Game.js

```diff
@@ -0,0 +1,409 @@
+import React from 'react';
+import './game.css';
+import Info from './Info.js';
+import PlayerChoice from './PlayerChoice';
+import Button from './Button';
+import ComputerChoice from './ComputerChoice';
+import Popup from './Popup';
+import EndGame from './EndGame';
+
+import winsound from '../audio/win.mp3';
+import losesound from '../audio/lose.mp3';
+import MusicButton from './MusicButton';
+import SoundButton from './SoundButton';
+import NewGameButton from './NewGameButton';
+
+
+class Game extends React.Component {
```

Comment:
it is god-like class. think about refactoring and splitting the logic into smaller pieces.